File: Assignment39_1.java
///////////////////////////////////////////////////////////////////////////////
// Function Name : Display
// Description   : Display stars using recursion
// Input         : ---
// Output        : *  *  *  *  *  *
// Author        : Raj Samir Jadhav
// Date          : 07/12/2025
///////////////////////////////////////////////////////////////////////////////
class Pattern
{
    static int i = 0;

    void Display()
    {
        if(i < 6)
        {
            System.out.print("*  ");
            i++;
            Display();            // Recursive call
        }
    }
}

class Assignment39_1
{
    public static void main(String[] args)
    {
        Pattern obj = new Pattern();
        obj.Display();
    }
}

File: Assignment39_1.c
#include <stdio.h>

///////////////////////////////////////////////////////////////////////////////
// Function Name : Display
// Description   : Display 6 stars using recursion
// Input         : ---
// Output        : *  *  *  *  *  *
// Author        : Raj Samir Jadhav
// Date          : 07/12/2025
///////////////////////////////////////////////////////////////////////////////
void Display()
{
    static int i = 0;

    if(i < 6)           // Base condition
    {
        printf("*  ");
        i++;
        Display();     // Recursive call
    }
}

int main()
{
    Display();
    return 0;
}
